src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/CommunicationRequestDetails.java

```diff
@@ -23,5 +23,6 @@ public class CommunicationRequestDetails {
     private CommunicationRequestTopic requestTopic;
     private String requestMessage;
     private CommunicationRequestReply requestReply;
+    private YesNo requestActioned;
```

Comment:
don't need this as replyHasBeenActioned has been created within CommunicationRequestReply (refactor your tribs-case-api code to use it here instead of in the request object.

Reply:
Done